src/ui/pages/config/ContentPagesContainer.js

```diff
@@ -86,8 +77,41 @@ export const mapDispatchToProps = dispatch => ({
   onClickViewPublishedPage: (page, domain, locale) => {
     window.open(`${domain}/${locale}/${page.code}.page`, '_blank');
   },
-  onExpandAll: () => dispatch(fetchPageTreeAll()),
+  onExpandAll: (currentPage) => {
+    dispatch(fetchPageTreeAll());
+    dispatch(setSelectedPage(currentPage));
+  },
   onCollapseAll: () => dispatch(collapseAll()),
+  onPageSearch: (value) => {
+    let queryString = convertToQueryString({
```

Comment:
what `queryString` does when `value` is falsy?

Reply:
Yeah I'll probably put it in the `if` block as well to be clear.